test/acceptance/tests.py

```diff
@@ -212,10 +215,32 @@ def do_staff_override(self, username, final_score=STAFF_AREA_SCORE.format(STAFF_
         self.staff_area_page.visit()
         self.staff_area_page.show_learner(username)
         self.staff_area_page.expand_learner_report_sections()
-        self.staff_area_page.assess("staff", self.STAFF_OVERRIDE_OPTIONS_SELECTED)
+        self.staff_area_page.staff_assess(self.STAFF_OVERRIDE_OPTIONS_SELECTED)
         self.staff_area_page.verify_learner_final_score(final_score)
 
 
+    def do_staff_assessment(self, number_to_assess=0, options_selected=OPTIONS_SELECTED):
+        """
+        Use staff tools to assess available responses.
+
+        Args:
+            number_to_assess: the number of submissions to assess. If not provided (or 0),
+                will grade all available submissions.
+        """
+        self.staff_area_page.visit()
+        self.staff_area_page.expand_staff_grading_section()
+        start_numbers = self.staff_area_page.available_checked_out_numbers
+        assessed = 0
+        while number_to_assess == 0 or assessed < number_to_assess:
+            continue_after = False if number_to_assess-1 == assessed else True
+            self.staff_area_page.staff_assess(options_selected, continue_after)
+            assessed += 1
+            new_numbers = (start_numbers[0]-assessed, start_numbers[1])
+            self.assertEqual(new_numbers, self.staff_area_page.available_checked_out_numbers)
```

Comment:
TODO: this isn't waiting long enough - I can see the correct numbers appear while the test is running, but it's failing with not-yet-updated numbers being reported.